descriptors/ModuleDescriptor-template.json

```diff
@@ -665,6 +665,22 @@
         }
       ]
     },
+    {
+      "id": "wrapper-pieces",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/orders/wrapper-pieces",
+          "permissionsRequired": ["orders.wrapper-pieces.collection.get"]
```

Comment:
Need to add module permissions for invoking storage wrapper pieces endpoints and for other endpoints like to get users tenants etc.